### PR TITLE
docs: update the triage and labels doc with the latest processes

### PR DESCRIPTION
**Pull request by ocombe**
## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

## What is the current behavior?
The doc for triage & labels is outdated.

Issue Number: #28058

## What is the new behavior?
The doc for triage & labels is updated with the latest processes

## Does this PR introduce a breaking change?

- [x] No

### CODE REVIEW

File: docs/TRIAGE_AND_LABELS.md [hunk 1]
```diff
@@ -65,24 +65,29 @@ What kind of problem is this?
...
 It is the caretaker's responsibility to assign `comp:  *` to each new issue as they come in.
+Issues that haven't been triaged can be found by selecting the issues with no milestone.
 
 If it's obvious that the issue or PR is related to a release regression, the caretaker is also responsible for assigning the `severity(5): regression` label to make the issue or PR highly visible.
 
 The primary triage should be done on a daily basis so that the issues become available for secondary triage without much of delay.
 
 The reason why we limit the responsibility of the caretaker to this one label is that it is likely that without domain knowledge the caretaker could mislabel issues or lack knowledge of duplicate issues.
 
+Once the primary triage is done, the ng-bot automatically adds the milestone `needsTriage`.
```
Comment (IgorMinar) [inline docs/TRIAGE_AND_LABELS.md]: is it ng-bot, angular-bot, angular-robot, or ng-robot? we seem to be using most of these. Could we agree on just one? ng-bot seems to be the best name (shorts). If that's fine then we should rename the github repo and any other doc instances that mention other name as well.

File: docs/TRIAGE_AND_LABELS.md [hunk 2]
```diff
@@ -105,10 +110,17 @@ These criteria are then used to calculate a "user pain" score as follows:
...
 This score can then be used to estimate the impact of the issue which helps with prioritization.
 
+Once the component's owner triage is done, the ng-bot automatically changes the milestone from `needsTriage` to `Backlog`.
+
 
 ## Triaging PRs
 
-Triaging PRs is the same as triaging issues, except that PRs have additional label categories that should be used to signal their state.
+Triaging PRs is the same as triaging issues, except that the labels `frequency: *` and `severity: *` are replaced by:
+- `type: *`
```
Comment (IgorMinar) [inline docs/TRIAGE_AND_LABELS.md]: type is not replaced. it's shared between issues and PRs. could you reword this please?

File: docs/TRIAGE_AND_LABELS.md [hunk 3]
```diff
@@ -161,3 +173,9 @@ Only issues with `cla:yes` should be merged into master.
 
 ### `aio: preview`
 Applying this label to a PR makes the angular.io preview available regardless of the author. [More info](../aio/aio-builds-setup/docs/overview--security-model.md)
+
+### `PR action: merge-assistance`
+This label can be added to let the caretaker know that the PR needs special attention.
+There should always be a comment added to the PR to explain why the caretaker's assistance is needed.
```
Comment (IgorMinar) [inline docs/TRIAGE_AND_LABELS.md]: can we define the format of the comment. I propose:

`merge-assistance: <explain what kind of assistance you need, and if not obvious why>`

**Review by IgorMinar — APPROVED** [review]
I suggested some changes, but lgtm overall.

File: docs/TRIAGE_AND_LABELS.md (hunk 1 above)
Comment (ocombe) [reply docs/TRIAGE_AND_LABELS.md]: The comp label is "ngbot", and the bot name itself is "ngbot" as well

**Review by kara — APPROVED** [review]
LGTM

**Pull request closed without merging**